BenadrylAutomation/src/main/java/com/jnj/actions/Action.java

```diff
@@ -1004,30 +1004,16 @@ public static void navigateBack() {
 	public static void navigateForward() {
 		driver.navigate().forward();
 	}
-
+	
 	/**
 	 * Function to verify page url
-	 * 
 	 * @param partialUrl pass partial url to verify
 	 */
 	public static void verifyPageUrl(String partialUrl) {
 		String currentUrl = driver.getCurrentUrl().trim();
-		extentInfoLog("Asserting current URL : ", currentUrl);
+        extentInfoLog("Asserting current URL : ", currentUrl);
```

Comment:
@zahinSJI format this file properly.

Reply:
Format is fine in my end
![image](https://github.com/sji-autobots/BenadrylAutomation/assets/107922907/c8543d84-9fdf-4c0a-b075-9b0f9da3a6c1)